app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -55,28 +55,29 @@ class MainActivity : AppCompatActivity() {
         setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
 
-        DatabaseManager.init(binding.root.context);
+        DatabaseManager.init(binding.root.context)
         CurrentProjectManager.loadFirstProject()
 
 
     }
 
     fun showMenuItems(items:Set<Int>){
+        val allActionMenuIds= setOf(R.id.action_add,R.id.action_delete_all_groups,R.id.action_show_hide_keybinds)
```

Comment:
Moved this here since it was the only place it was used, so instead of making it private it seemed a better to put it withing the function itself.
source: https://kotlinlang.org/docs/functions.html#parameters

Reply:
agreed.